<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>publish</class>
 <widget class="QDialog" name="publish">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New publication</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #4f7f8f;  /* Fondo azul verdoso para la ventana */
    color: #333333;  /* Texto oscuro para el fondo azul */
    font-family: &quot;Arial&quot;, sans-serif;
    font-size: 16px;  /* Tamaño de fuente más grande para toda la aplicación */
}

QLabel {
    color: #a8d8a8;  /* Verde claro para el texto de los labels */
    font-size: 18px;  /* Tamaño de fuente más grande en los labels */
    font-weight: bold;
}

QPushButton {
    background-color: #6baed6;  /* Azul suave para los botones */
    color: white;
    border: 2px solid #4f7f8f;  /* Borde del botón en color azul verdoso */
    border-radius: 5px;
    padding: 10px 20px;
    font-size: 16px;  /* Tamaño de fuente más grande en los botones */
}

QPushButton:hover {
    background-color: #4f7f8f;  /* Al pasar el ratón se vuelve más oscuro */
    color: white;
    border-color: #6baed6;
}

QPushButton:pressed {
    background-color: #3b6e7d;  /* Cuando se presiona, un tono más oscuro */
}

QLineEdit {
    background-color: #e0f7fa;  /* Fondo suave de color verde claro para los inputs */
    border: 1px solid #4f7f8f;  /* Borde verde para los campos de texto */
    color: #333333;
    padding: 8px;
    border-radius: 5px;
    font-size: 16px;  /* Tamaño de fuente más grande en los inputs */
}

QCheckBox {
    color: #a8d8a8;  /* Verde claro para el texto del checkbox */
    font-size: 16px;  /* Tamaño de fuente más grande en el checkbox */
}

QRadioButton {
    color: #a8d8a8;  /* Verde claro para el texto del radiobutton */
    font-size: 16px;  /* Tamaño de fuente más grande en el radiobutton */
}

QComboBox {
    background-color: #e0f7fa;  /* Fondo verde claro */
    border: 1px solid #4f7f8f;  /* Borde del combobox */
    color: #333333;
    font-size: 16px;  /* Tamaño de fuente más grande en el combobox */
    padding: 8px;
}

QComboBox:hover {
    border-color: #6baed6;  /* Borde azul claro cuando se pasa el ratón */
}

QScrollBar {
    background-color: #e0f7fa;  /* Fondo verde claro para la barra de desplazamiento */
    border: 1px solid #4f7f8f;  /* Borde azul verdoso */
}

QScrollBar::handle {
    background-color: #6baed6;  /* Color del control de la barra de desplazamiento */
}

QScrollBar::handle:hover {
    background-color: #4f7f8f;  /* Color más oscuro cuando se pasa el ratón */
}

QTabWidget::pane {
    border: 1px solid #4f7f8f;  /* Borde para los paneles de las pestañas */
}

QTabBar::tab {
    background-color: #6baed6;  /* Fondo azul claro para las pestañas */
    color: white;
    padding: 12px;
}

QTabBar::tab:selected {
    background-color: #4f7f8f;  /* Fondo azul oscuro para la pestaña seleccionada */
}

QSlider {
    background-color: #e0f7fa;  /* Fondo verde claro */
    border: 1px solid #4f7f8f;  /* Borde azul verdoso */
}

QSlider::handle {
    background-color: #6baed6;  /* Color del control deslizante */
    border: 1px solid #4f7f8f;
}


QTabWidget::pane {
    border: 1px solid #4f7f8f;  /* Borde para los paneles de las pestañas */
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="categoryComboBox"/>
    </item>
    <item>
     <widget class="QLineEdit" name="title">
      <property name="inputMask">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Title</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="upload_path"/>
      </item>
      <item>
       <widget class="QPushButton" name="search_path">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="confirm">
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
